res/controllers/Gemini-G4V-scripts.js

```diff
@@ -0,0 +1,1802 @@
+            this.padConnections["s"+(sampleGroup+3)+"_play_indicator"].activate();
+            this.padConnections["s"+(sampleGroup+4)+"_play_indicator"].activate();
+            this.padConnections["s"+(sampleGroup+5)+"_play_indicator"].activate();
+            this.padConnections["s"+(sampleGroup+6)+"_play_indicator"].activate();
+            this.padConnections["s"+(sampleGroup+7)+"_play_indicator"].activate();
+        }
+        if (set === 4) { // Loop Roll mode
+            this.padConnections["beatlooproll_0.125_activate"].activate();
+            this.padConnections["beatlooproll_0.25_activate"].activate();
+            this.padConnections["beatlooproll_0.5_activate"].activate();
+            this.padConnections.beatlooproll_1_activate.activate();
+            this.padConnections.beatlooproll_2_activate.activate();
+            this.padConnections.beatlooproll_4_activate.activate();
+            this.padConnections.beatlooproll_8_activate.activate();
+            this.padConnections.beatlooproll_16_activate.activate();
+        }
+        if (set === 5) { // Manual loop mode
+            this.padConnections.loop_in.activate();
+            this.padConnections.loop_out.activate();
+            this.padConnections.loop_enabled.activate();
+        }
+        if (set === 6) {    // Beat Jump mode
+            // Leds off - no connections
+            this.leds.pad1.set(0);
+            this.leds.pad2.set(0);
+            this.leds.pad3.set(0);
+            this.leds.pad4.set(0);
+            this.leds.pad5.set(0);
+            this.leds.pad6.set(0);
+            this.leds.pad7.set(0);
+            this.leds.pad8.set(0);
+        }
+
+        // Sets pad mode
+        this.padMode = set;
+    };
+
+    // Shutdown the deck
+    this.shutdown = function() {
+        var myObj = this;
+        Object.keys(this.leds).forEach(function(element, _key) { myObj.leds[element].set(0); });
+    };
+
+    //////////////////////////////////////////////////////////////////////////////
+    // Call back functions for connecting Mixxx controls to controller controls //
+    //////////////////////////////////////////////////////////////////////////////
+    this.cbPlay = function(value, group, _control) {
+        g4v.decks[group.substring(8, 9)].leds.play.set(value === 0 ? 0 : 1);
+    };
+
+    this.cbCue = function(value, group, _control) {
+        g4v.decks[group.substring(8, 9)].leds.cue.set(value === 0 ? 0 : 1);
+    };
+
+    this.cbSync = function(value, group, _control) {
+        g4v.decks[group.substring(8, 9)].leds.sync.set(value === 0 ? 0 : 1);
+    };
+
+    this.cbKeylock = function(value, group, _control) {
+        g4v.decks[group.substring(8, 9)].leds.keylock.set(value === 0 ? 0 : 1);
+    };
+
+    this.cbPad1 = function(value, group, _control) {
+        g4v.decks[(group.substring(1, 2) === "C") ? group.substring(8, 9) : Math.ceil(group.substring(8, (group.substring(9, 10) === "]" ? 9 : 10))/16)].leds.pad1.set(value === 0 ? 0 : 1);
```

Comment:
this looks like you want to use `script.deckFromGroup`.

Reply:
Most probable. I was not aware of it :)